### PR TITLE
Fix panel

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: src/components/file-manager/file-manager-slide.tsx [hunk 1]
```diff
@@ -358,13 +373,13 @@ const FileManagerSlide = ({
...
                         <TableIcon src={ICON_LOGO} />
                       </Td>
-                      <Td>{name}</Td>
-                      <Td>{handle}</Td>
-                      <Td>{modifiedAt}</Td>
-                      <Td>{size} FILES</Td>
+                      <Td onClick={() => filePanel(handle)}>{name}</Td>
+                      <Td onClick={() => filePanel(handle)}>{handle}</Td>
+                      <Td onClick={() => filePanel(handle)}>{modifiedAt}</Td>
+                      <Td onClick={() => filePanel(handle)}>{size} FILES</Td>
```
Comment (EdmundMai) [inline src/components/file-manager/file-manager-slide.tsx]: remove these and add `onClick` to the surrounding `Tr`

File: src/components/file-manager/file-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+`;
+
+const FilePanel = ({ handle, download, closeButton }) => {
+  const [file, setFile] = useState({
+    name: "HR Stuff",
+    handle: "0x0x0x",
+    modifiedAt: "01/03/2019",
+    size: 40,
+    icon: ICON_LOGO
+  });
```
Comment (EdmundMai) [inline src/components/file-manager/file-panel.tsx]: this should be passed by the props from the file-manager-slide when the user clicks the `Tr`

File: src/components/file-manager/file-panel.tsx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+      .catch(() =>
+        setFile({
+          name: "HR Stuff",
+          handle: "0x0x0x",
+          modifiedAt: "01/03/2019",
+          size: 40,
+          icon: ICON_LOGO
+        })
+      );
+  }, []);
```
Comment (EdmundMai) [inline src/components/file-manager/file-panel.tsx]: remove, no request needed

File: src/components/file-manager/file-panel.tsx [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #687892;
+  margin-left: 5px;
+  float: left;
+`;
+
+const FilePanel = ({ handle, download, closeButton }) => {
```
Comment (EdmundMai) [inline src/components/file-manager/file-panel.tsx]: i had a PR for this before but reverted it. i thought we wanted to remove the hamburger?

Comment (tenisakb) [reply src/components/file-manager/file-panel.tsx]: fixed

File: src/components/file-manager/file-panel.tsx (hunk 3 above)
Comment (tenisakb) [reply src/components/file-manager/file-panel.tsx]: fixed

File: src/components/file-manager/file-panel.tsx (hunk 2 above)
Comment (tenisakb) [reply src/components/file-manager/file-panel.tsx]: fixed

File: src/components/file-manager/file-manager-slide.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/file-manager/file-manager-slide.tsx]: fixed

**Pull request closed without merging**